traffic_ops/traffic_ops_golang/invalidationjobs/invalidationjobs.go

```diff
@@ -146,20 +196,40 @@ RETURNING job.asset_url,
            WHERE deliveryservice.id=job.job_deliveryservice
           ) AS delivery_service,
           job.id,
-          job.keyword,
-          job.parameters,
-          job.start_time
+		  ttl_hr,
+          start_time,
+		  invalidation_type
 `
 
+// Deprecated, only to be used with versions below 4.0
 const putInfoQuery = `
 SELECT job.id AS id,
        tm_user.username AS createdBy,
        job.job_user AS createdByID,
        job.job_deliveryservice AS dsid,
        deliveryservice.xml_id AS dsxmlid,
        job.asset_url AS assetURL,
-       job.parameters,
+       CONCAT('TTL:', ttl_hr, 'h') AS parameters,
+       job.start_time AS start_time,
+       origin.protocol || '://' || origin.fqdn || rtrim(concat(':', origin.port), ':') AS OFQDN
+FROM job
+INNER JOIN origin ON origin.deliveryservice=job.job_deliveryservice AND origin.is_primary
+INNER JOIN tm_user ON tm_user.id=job.job_user
+INNER JOIN deliveryservice ON deliveryservice.id=job.job_deliveryservice
+WHERE job.id=$1
+`
+
+// Almost the same as putInfoQuery, but returns appropriate values for API 4.0+
+const putInfoQueryV40 = `
```

Comment:
We should probably call this V4 instead of V40, because we'll likely just update this query with new fields if they're ever added.